### ISSUE TITLE
Question about Function： ValidateClusters

### ISSUE THREAD

**Issue by illidandan** [satijalab/seurat#707]
I’m using Seurat and I tried to validate the legitimacy of my clusters using “ValidateClusters” Function.

But I found that no matter how much pc.use I set, the new results is always the same as the old result.

So I am wondering if this Function works.

My script is as below 
```R
tnbc <- FindClusters(object = tnbc, reduction.type = "pca", dims.use = 1:20, resolution = 1.5, save.SNN = TRUE)

write.table(tnbc@ident,file=”f1.txt”,quote=F,sep=”\t”)

tnbc<- ValidateClusters(tnbc, pc.use = 1:20)

write.table(tnbc@ident,file=”f2.txt”,quote=F,sep=”\t”)

tnbc<- ValidateClusters(tnbc, pc.use = 1:30)
.
write.table(tnbc@ident,file=”f3.txt”,quote=F,sep=”\t”)

tnbc<- ValidateClusters(tnbc, pc.use = 1:2)

write.table(tnbc@ident,file=”f4.txt”,quote=F,sep=”\t”)
```
the output files are all same to each other.

**Comment by andrewwbutler** [comment]
Hi,

It is possible that you have very distinct clusters that are very easy to distinguish based on the classifier that's being built and thus never merging any clusters. Are you seeing any output from the function? You could also try adjusting the accuracy or connectivity thresholds.

**Comment by illidandan** [comment]
Thanks for replying. 

The output of this function is “100% complete --- started with 22 clusters, 22 clusters remaining”.
I tried to adjust the accuracy thresholds to 99 and it returns a same result.
But when I tried to set the min.connectivity to 0.001,the Function load the "lattice" package and print 
"6% complete --- merge clusters 5 and 7, classification accuracy of 0.7865"
I think it might be a bug.
The Function does not work unless user set the min.connectivity arguement in the Function.
